### ISSUE TITLE
Divide notifications from Notification Widget to keep the forecast visible on fisrt sight

### ISSUE THREAD

**Issue by Moostek** [breezy-weather/breezy-weather#797]
### Describe your idea

Hello, maybe it is not a bug, but for me it is a cumbersome solution. I will try to explain. I use the Notification Widget with 5-Day forecast style. Normally it shows the forecast well (except on bug with Locksreen mode I have reported earlier), but when I get Daily forecast at 7 am, the Notification Widget (with 5-Day forecast) is consolidated with Daily forecast and the Notification Widget is reduced in what it shows (only actual temperature), so since then I can not see 5-day forecast on first sight, I have to expand Breeze Weather stripe to show it. Only when I delete the Daily forecast, the original Notification Widget with 5-Day forecast is displayed again. Same it goes with other Breeze Weather notifications, like the one that tells me that the synchronization failed. Again, I have to delete the notification to get the original Notification Widget with 5-Day forecast. 

**I suggest to separate each Breeze Weather notification, i.e. separate it from Notification Widget to keep 5-Day forecast visible permanently.** Somebody might like it as it is now. So I suggest to make it selectable via Settings, of course. 

Thank you. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to a question covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document, or a feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 5.1.3 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Can we do that? Isn't Android grouping all notifications by app? Can you show an example of screenshot of an open source where notifications from the same app are not grouped?

**Comment by Moostek** [comment]
Right now I have no such notification from open-source app. But I can tell you that "Etar" calendar (https://github.com/Etar-Group/Etar-Calendar) give me every notification (Event) in separate item, it is not grouped under Etar folder or so. And as I remember I got separate notifications from from other open-source apps. Plus I can confirm, that another weather app (it is not FOSS one, but fair one without 0 trackers etc.) doing the same: Notification Widget is standalone, so other notifications are separated from Notification Widget. I really think that it is possible.

**Comment by papjul** [comment]
I will check

**Comment by Moostek** [comment]
Cool, thank you.

**Comment by min7-i** [comment]
> But I can tell you that "Etar" calendar give me every notification (Event) in separate item...

At least on Android 14 Etar's notifications are grouped the same way as the notifications of Breezy Weather.

However, it might help looking at [tasks](https://github.com/tasks/tasks) which has an option to (un)group notifications. Ungrouping notifications looks like this:
| setting | result |
| --- | --- |
| ![tasks_notification_settings](https://github.com/breezy-weather/breezy-weather/assets/97251923/5e04ec1a-6035-4d05-8358-b78fdc2ecdc6) | ![tasks_notification_drawer](https://github.com/breezy-weather/breezy-weather/assets/97251923/7b1092d8-0a36-431f-8c58-e1254a113786) |

**Comment by papjul** [comment]
They still look grouped to me? What does it look like when they are?

The notification with the higher importance (for example notification an alert) will still be on top, while the notification-widget below collapsed.

**Comment by min7-i** [comment]
> The notification with the higher importance (for example notification an alert) will still be on top, while the notification-widget below collapsed.

Correct, this will probably not change the collapsed state of the notification.

Below are screenshots showing the difference between the "bundle notifications" being enabled and disabled. For comparison, I also added some notifications of Breezy Weather.

| Android version | grouped | ungrouped |
| --- | --- | --- |
| A14 | ![a14_grouped](https://github.com/breezy-weather/breezy-weather/assets/97251923/2237616b-50d4-4c20-b737-b9eba3e86c6b) | ![a14_ungrouped](https://github.com/breezy-weather/breezy-weather/assets/97251923/5474bded-706f-43d5-8310-3c2161f936a9) |
| A9 | ![A9_grouped](https://github.com/breezy-weather/breezy-weather/assets/97251923/c0e9fbb8-5901-431f-872a-8a68f84f37e0) | ![A9_ungrouped](https://github.com/breezy-weather/breezy-weather/assets/97251923/01229de7-e45c-4505-b793-368a08743ab4) |

**Comment by papjul** [comment]
So, we have no change to the collapsed status, only a minor difference of display? Doesn't look worth it.
Also, grouped notifications are here to avoid having one application taking all the space for itself, so I don’t think we should workaround this.

I believe notification-widget was made because of the lack of widgets on lockscreen, but [there is some hope](https://www.androidauthority.com/android-15-lock-screen-widgets-3402342/).

I also heard about [SmartSpacer](https://github.com/KieronQuinn/Smartspacer), not sure if it can help here? There is a weather plugin here: https://github.com/pacjo/SmartspacerPlugins

Can we proceed to close as "not possible"?
